test/test-lib/fake-device.ts

```diff
@@ -18,6 +18,11 @@ export async function getState(
 	deviceUuid: string,
 	stateVersion: 'v3',
 ): Promise<StateV3>;
+export async function getState(
```

Comment:
Nice one @thgreasi, I had tried the `as const` but didn't think to add another overload to the function